docs/guide/custom_policies.md

```diff
@@ -119,40 +119,168 @@ class Kernel extends BaseKernel
 
+```php
+<?php
+
+namespace App\Security\Limitation\Mapper;
+
+use eZ\Publish\API\Repository\Values\User\Limitation;
+use EzSystems\EzPlatformAdminUi\Translation\Extractor\LimitationTranslationExtractor;
+use EzSystems\RepositoryForms\Limitation\LimitationFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+
+class CustomLimitationFormMapper implements LimitationFormMapperInterface
+{
+    public function mapLimitationForm(FormInterface $form, Limitation $data)
+    {
+        $form->add('limitationValues', CheckboxType::class, [
+            'label' => LimitationTranslationExtractor::identifierToLabel($data->getIdentifier()),
+            'required' => false,
+            'data' => $data->limitationValues['value'],
+            'property_path' => 'limitationValues[value]'
+        ]);
+    }
+
+    public function getFormTemplate()
+    {
+        return '@ezdesign/limitation/custom_limitation_form.html.twig';
+    }
+
+    //TODO: public function filterLimitationValues(Limitation $limitation)
+}
+```
+
+```html+twig
+{# templates/themes/standard/limitation/custom_limitation_form.html.twig #}
+{{ form_label(form.limitationValues) }}
+{{ form_widget(form.limitationValues) }}
+```
+
 Next, register the service with the `ez.limitation.formMapper` tag and set the `limitationType` attribute to the Limitation type's identifier:
 
 ```yaml
 App\Security\Limitation\Mapper\CustomLimitationFormMapper:
     arguments:
         # ...
     tags:
-        - { name: 'ez.limitation.formMapper', limitationType: 'Custom' }
+        - { name: 'ez.limitation.formMapper', limitationType: 'CustomLimitation' }
 ```
 
 If you want to provide human-readable names of the custom Limitation values, you need to implement [`EzSystems\EzPlatformAdminUi\Limitation\LimitationValueMapperInterface`](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/lib/Limitation/LimitationValueMapperInterface.php).
 
+```php
+<?php
+
+namespace App\Security\Limitation\Mapper;
+
+use eZ\Publish\API\Repository\Values\User\Limitation;
+use EzSystems\EzPlatformAdminUi\Limitation\LimitationValueMapperInterface;
+
+class CustomLimitationValueMapper implements LimitationValueMapperInterface
+{
+    public function mapLimitationValue(Limitation $limitation)
+    {
+        return $limitation->limitationValues['value'];
+    }
+}
+```
+
 Then register the service with the `ez.limitation.valueMapper` tag and set the `limitationType` attribute to Limitation type's identifier:
 
 ```yaml
 App\Security\Limitation\Mapper\CustomLimitationValueMapper:
     arguments:
         # ...
     tags:
-        - { name: 'ez.limitation.valueMapper', limitationType: 'Custom' }
+        - { name: 'ez.limitation.valueMapper', limitationType: 'CustomLimitation' }
 ```
 
-If you want to completely override the way of rendering custom Limitation values in the role view,
```

Comment:
This is not optional but mandatory

`An exception has been thrown during the rendering of a template ("Could not find a block for CustomLimitation: ez_limitation_customlimitation_value.").`

Reply:
As soon as a value mapper exist, the template is mandatory.